scripts/exglobal_prep_ocean_obs.py

```diff
@@ -95,8 +94,12 @@ def run_netcdf_to_ioda(obsspace_to_convert):
                 outputFilename = f"gdas.t{cyc}z.{obs_space_name}.{PDY}{cyc}.nc4"
                 obsprepSpace['output file'] = outputFilename
 
-                iodaYamlFilename = obsprepSpaceName + '2ioda.yaml'
-                save_as_yaml(obsprepSpace, iodaYamlFilename)
+                # Skip in situ IODA conversion for now
```

Comment:
I've been thinking about how to decide on how to handle bufr vs. netcdf and I wonder if it would make more sense to give them explicit fields in the obsprep_config.yaml, something like
`converter: bufr` (or `netcdf`)
One thing would be good to know: do the two converters take the same format input yaml?

Reply:
@AndrewEichmann-NOAA we have been discussing your idea.
`bufr` config file is json.